paimon-flink/paimon-flink-common/src/test/java/org/apache/paimon/flink/ReadWriteTableITCase.java

```diff
@@ -1570,6 +1570,85 @@ public void testUpdateWithoutPrimaryKey(WriteMode writeMode) throws Exception {
                 .satisfies(AssertionUtils.anyCauseMatches(UnsupportedOperationException.class));
     }
 
+    @ParameterizedTest
+    @EnumSource(CoreOptions.MergeEngine.class)
+    public void testDeleteWithPrimaryKey(CoreOptions.MergeEngine mergeEngine) throws Exception {
```

Comment:
Better to add Delete statement comment above this test (like update statement tests).